### ISSUE TITLE
Runtime error: mat1 dim 1 must match mat2 dim 0 

### ISSUE THREAD

**Issue by p06p05** [maxjcohen/transformer#47]
Hey,
Thanks for providing code for the prediction of time series using Transformer. I am using Oze energy challenge dataset with 7500 samples. I am sticking at following lines:
# Propagate input
            netout = net(x.to(device))

            # Comupte loss
            loss = loss_function(y.to(device), netout)

probably I am not able to set d_input,d_model and d_output parameters correctly. please help me to run this code on said dataset.
